<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerDialog</class>
 <widget class="QDialog" name="customerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1151</width>
     <height>751</height>
    </rect>
   </property>
   <property name="title">
    <string>Customer Page</string>
   </property>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>441</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_info">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>621</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>All The Fleet Information below</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>675</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cbBox_lb">
       <item>
        <property name="text">
         <string>Embraer Phenom 300</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cessna Citation XLS</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_ms">
       <item>
        <property name="text">
         <string>Citation Sovereign </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Citation Latitude</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_sms">
       <item>
        <property name="text">
         <string>Bombardier Challenge 350 </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cessna Citation Longtitude</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBox_large">
       <item>
        <property name="text">
         <string>Bombardier challenge 650</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gulf Stream G450</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bombardier Global 5000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bombardier Global 6000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bombardier Global 7500</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_choose">
       <property name="text">
        <string>Choose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_fleet">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_checkout">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>401</width>
      <height>681</height>
     </rect>
    </property>
    <property name="title">
     <string>Check out (you can can not exceeds 3 fleets per purchase)</string>
    </property>
    <widget class="QLabel" name="lbl_error">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_3">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>185</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Price Packages</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_finalcialpackage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>371</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>630</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>CHECK OUT </string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>142</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_remove1">
         <property name="text">
          <string>remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_remove2">
         <property name="text">
          <string>remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_remove3">
         <property name="text">
          <string>remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="lbl_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>391</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbtn_25hrs">
        <property name="text">
         <string>Travel &lt; 25 hours per year </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_50_once">
        <property name="text">
         <string>Travel &gt; 50 hours per year and prefer once payment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_50_12">
        <property name="text">
         <string>Travel &gt; 50 hours per year and prefer monthly payment </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>651</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Light Cabin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mid-size Cabin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Super mid-size cabin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Large</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
